src/src/rx_main.cpp

```diff
@@ -1260,12 +1260,16 @@ void reset_into_bootloader(void)
     Serial.println("Jumping to Bootloader...");
     delay(100);
 
-#if BOOTLOADER_DATA_EXCHANGE_ENABLED
+    /** Write command for firmware update.
```

Comment:
This is not used by the bootloaders yet.
Bootloader will overwrite values in RAM if not supported so this does not brake old bootloaders functionality.